src/main/java/com/mailjet/client/transactional/SendEmailsRequest.java

```diff
@@ -0,0 +1,61 @@
+package com.mailjet.client.transactional;
+
+import com.google.gson.FieldNamingPolicy;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.mailjet.client.MailjetClient;
+import com.mailjet.client.MailjetRequest;
+import com.mailjet.client.MailjetResponse;
+import com.mailjet.client.errors.MailjetException;
+import com.mailjet.client.resource.Emailv31;
+import com.mailjet.client.transactional.response.SendEmailsResponse;
+import lombok.Builder;
+import lombok.Builder.Default;
+import lombok.Singular;
+
+import java.util.List;
+
+@Builder
+public class SendEmailsRequest {
+
+    /**
+     * Adds message to sent to the request
+     * You can send up to 50 messages per batch
+     */
+    @Singular
+    private List<TransactionalEmail> messages;
+
+    /**
+     * If set to true, messages will be send to the Mailjet API for verification
+     * But the actual emails will not be send
+     */
+    private boolean sandboxMode;
+
+    @Default
+    public boolean AdvanceErrorHandling = true;
+
+    /**
+     * Represents a method to send multiple transactional emails
+     * @param mailjetClient the Mailjet client that will be used to send messages
+     * @return A response with sent messages information, or error information
+     * @throws MailjetException in case of communication error
```

Comment:
What does "communication error" embody exactly? What if the user is not allowed to send? What if the sending limit is reached? What if the user tries to send more than 50 emails? Those would all lead to exceptions as well, wouldn't they?

Reply:
In general, communication error means an error in HTTP stack - no internet connection, internal server error, or in general Response status code is not 2xx or 400.

If the request was successfully delivered to the API endpoint but failed validation on the server-side, like, limits validation, the regular response will be returned with the "error" status of the messages. 

Ofc some of these checks could be done beforehand, like, checking the email's amount, but I don't see a big value in ti - anyway, there are some checks that could be done on the server-side only.
Anyway, will update this doc

Reply:
updated, added handling for the 4xx status code.